source/System.cpp

```diff
@@ -1043,6 +1052,21 @@ void System::LoadObjectHelper(const DataNode &node, StellarObject &object, bool
 		object.speed = 360. / node.Value(1);
 	else if(key == "offset" && hasValue)
 		object.offset = node.Value(1);
+	else if(key == "visibility" && hasValue)
+	{
+		object.trueDistanceInvisible = node.Value(1);
+		if(node.Size() > 2)
```

Comment:
I think we usually are not so strict with it (so you can use >= 2 here and the same logic applies below)

Reply:
that does not make sense?
it has to have at least 3 values, so >= 2 makes no sense to me

Reply:
>= 3 ~~dont be picky uk thats what I meant~~

Reply:
it doesnt make a difference but here you go